test/built-ins/Array/from/calling-from-valid-1.js

```diff
@@ -0,0 +1,24 @@
+// Copyright 2015 Microsoft Corporation. All rights reserved.
+// This code is governed by the license found in the LICENSE file.
+
+/*---
+description: Calling from with a valid map function without thisArg
+es6id: 22.1.2.1
+---*/
+
+var array = [ 0, 1, -2, 4, -8, 16, -32, 64, -128, 256, -512, 1024 ];
+var arrayIndex = -1;
+var globalThis = this;

```

Comment:
The approach preferred in recent tests involving callback-supporting APIs is to include a dedicated test for strict mode (using the `onlyStrict` [frontmatter flag](https://github.com/tc39/test262/blob/d62f3a82b97dba22a0be2798fbe34548e96a439f/CONTRIBUTING.md#flags)) and a separate test for non-strict mode (using the `noStrict` frontmatter flag). Each of these tests can be more explicit about expectations for the `thisValue`. This protects against false positives where the value of `this` may be incorrect in the global scope.